tests/load/test_load_variant.py

```diff
@@ -58,12 +59,19 @@ def test_load_vep97_parsed_variant(
     adapter.load_variant(variant_obj=parsed_vep97_annotated_variant)
 
     # THEN the variant is loaded with the fields correctly parsed
+    # revel score
     variant = adapter.variant_collection.find_one()
     assert isinstance(variant["revel_score"], float)
 
+    # conservation fields
     for key, value in variant["conservation"].items():
         assert value == ["NotConserved"]
 
+    # clinvar fields
+    assert isinstance(variant["clnsig"][0]["accession"], int)
+    assert variant["clnsig"][0]["value"] in REV_CLINSIG_MAP  # can be str or int
+    assert variant["clnsig"][0]["revstat"] in REV_CLINSIG_MAP  # str
```

Comment:
I don't like this, I'll change it

Reply:
Yes, in particular the latter I would say is wrong? We dont really encode the revstats, do we? They go through unadjusted.

Reply:
Yes, I meant the latter!